sentry_sdk/integrations/dramatiq.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import absolute_import
+
+from dramatiq import Middleware, get_broker
+
+from sentry_sdk import Hub
+from sentry_sdk.integrations import Integration
+from sentry_sdk.utils import capture_internal_exceptions, event_from_exception
+
+
+class DramatiqIntegration(Integration):
+    """Automatically integrates the current Dramatiq broker with
+    Sentry.  If you use this, then make sure you call sentry_sdk.init
+    *after* setting up your broker, otherwise you can use the Sentry
+     middleware in this module directly.
+    """
+
+    identifier = "dramatiq"
+
+    def install(self):
+        get_broker().add_middleware(Sentry())
```

Comment:
I'm not faimilar with Dramatiq at all, but note that integrations have to be independent of the order stuff is configured.

For Flask and Django that for example means we can't just "get the current app" and attach stuff to it, because at the point where the integration is installed, the app might not yet exist:

```
sentry_sdk.init(...)
app = Flask(...)
```

or maybe some code later decides to register a new app as the current one.

Generally we try to hook into frameworks such that we capture everything from all instances that might be created in the future. In Flask this works fine because there are global signal handlers for this. In celery likewise. In Django and Sanic we just have to do excessive monkeypatching but it's worth the gain of not debugging weird import-order problems.